governance/votes/Executive vote - May 18, 2022.md

```diff
@@ -0,0 +1,78 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit MKR Vesting Streams
+
+As per their successful [MKR budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP17), a total of **541.47 MKR** will be distributed to the [Sustainable Ecosystem Scaling Core Unit (SES-001)](https://mips.makerdao.com/mips/details/MIP39c2SP10) at [0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6](http://etherscan.io/address/0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6) (SES-001's Auditor Wallet) if this executive proposal passes.
+
+### Foundational L2 Work
+
+The [Maker Teleport](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) initiative, [previously known as Maker Wormhole](https://forum.makerdao.com/t/maker-wormhole-new-name-maker-teleport/15115), will be bringing canonical DAI capability onto L2 domains. As part of this development, it is necessary to introduce a new module called `DssCure` and update the existing `End` module to interact with it.
+
+In the event of an [Emergency Shutdown](https://docs.makerdao.com/smart-contract-modules/shutdown), the `DssCure` module ensures a fair distribution to DAI holders by calculating how much debt needs to be reduced where there is pre-minted, unused DAI sitting in the `Vat`. Discounting these pre-minted, out-of-circulation DAI is necessary for the correct calculation of the outstanding DAI---otherwise, the artificial, increased debt resulting from factoring in the pre-minted DAI will result in DAI holders receiving less collateral than they are entitled to.
+
+`DssCure` sits between the `Vat` and `End` so that `End` receives the correct amount of debt.
+
+The following additions and changes will take place if this executive proposal passes:
+- A new `DssCure` module will be added to the Maker Protocol.
+- The `MCD_END` module will be replaced by a new version that interacts with `DssCure`.
+- The existing ESM contract will be made to point to the new `MCD_END`.
+
+For more details, please read [this forum post by the Protocol Engineering Core Unit (PE-001)](https://forum.makerdao.com/t/wednesday-18th-may-executive-dsscure-technical-enhancement/15175).
+
+### New [Flash Mint Module](https://manual.makerdao.com/module-index/module-flash-mint-module)
+
+As described in this [forum post](https://forum.makerdao.com/t/wednesday-18th-may-executive-flash-mint-module-technical-enhancement/15176) if this executive proposal passes the following changes will occur:
+* A new Flash Mint Module will be deployed and authorized.
+* The new module will have no fee parameter and charge no fee in order to save on gas.
+* The Debt Ceiling will be set to 250M on the new module and reduced to 250M on the existing module to maintain 500M total.
+* Both modules will be available in the chainlog as MCD_FLASH_LEGACY and MCD_FLASH.
```

Comment:
Maybe we want to clarify that eventually the old module will be removed and the new one will be the only active.